install/installing-on-linux.md

```diff
@@ -69,10 +69,10 @@ Now create a `.env` config file in the base of repository. Make sure that the da
 
 ```
 DB_HOST=localhost
-DB_NAME=username
-DB_PASS=password
+DB_NAME=ushahidi
 DB_TYPE=MySQLi
-DB_USER=ushahidi
+DB_USER=username
+DB_PASS=password
 ```
```

Comment:
In that same vein, regarding the GRANT command, shouldn't 'set-a-custom-password-here' be the same as 'password' in DB_PASS or am I miss understanding?

GRANT ALL ON ushahidi.* to ushahidi@localhost IDENTIFIED BY 'set-a-custom-password-here';